playbooks/templates/dev-install_net_config.yaml.j2

```diff
@@ -26,6 +26,14 @@ network_config:
   - type: interface
     name: dummy0
     nm_controlled: true
+{% for ip in tunnel_remote_ips | default([]) %}
+  - type: ovs_tunnel
+    name: "tun-ctlplane-{{ ip | to_uuid }}"
+    tunnel_type: vxlan
+    ovs_options:
+      - remote_ip={{ ip }}
+      - key=100
+{% endfor %}
 - type: ovs_bridge
   name: br-hostonly
```

Comment:
We probably ought to change the name of this bridge eventually.

Reply:
ack